### PR TITLE
fix: keepkey swaps from RUNE

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Bumps `chain-adapters` to latest, see https://github.com/shapeshift/lib/pull/1231 for more info on what this does and why.

tl;dr uses a `MsgDeposit` message that `hdwallet-keepkey` accepts and won't throw, and that can be signed by the KeepKey firmware, making swaps from RUNE on KeepKey great (unfortunately not again).

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

None, can't break what's already broken

## Testing

- Connect a KeepKey
- Ensure THORChain swaps from RUNE are working
- As a sanity check, ensure swaps to RUNE are also working, though the lib PR doesn't actually touch this

- Expected/min. amounts aren't a concern of this PR

### Engineering

- ☝🏽 

### Operations

- ☝🏽 

## Screenshots (if applicable)

image
image
image
image
image

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #4186** <a href="https://app.graphite.dev/github/pr/shapeshift/web/4186" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/4186?utm_source=stack-comment).

**Review by kaladinlight — APPROVED** [review]
